sdk/kubernetesconfiguration/Azure.ResourceManager.KubernetesConfiguration/api/Azure.ResourceManager.KubernetesConfiguration.netstandard2.0.cs

```diff
@@ -1,23 +1,23 @@
 namespace Azure.ResourceManager.KubernetesConfiguration
 {
-    public partial class ExtensionCollection : Azure.ResourceManager.ArmCollection, System.Collections.Generic.IAsyncEnumerable<Azure.ResourceManager.KubernetesConfiguration.ExtensionResource>, System.Collections.Generic.IEnumerable<Azure.ResourceManager.KubernetesConfiguration.ExtensionResource>, System.Collections.IEnumerable
+    public partial class KubernetesClusterExtensionCollection : Azure.ResourceManager.ArmCollection, System.Collections.Generic.IAsyncEnumerable<Azure.ResourceManager.KubernetesConfiguration.KubernetesClusterExtensionResource>, System.Collections.Generic.IEnumerable<Azure.ResourceManager.KubernetesConfiguration.KubernetesClusterExtensionResource>, System.Collections.IEnumerable
```

Comment:
Please see my comments in the https://apiview.dev/Assemblies/Review/56be882ab2fa4bfe9d1d5611c5e80d51

Reply:
Replied in apiview. Let me know what you think.